File: 2_course/ЯПВУ/Laptev_AV1/Лаптев Александр/LaptevAV-2_3.rb
# encoding utf-8
puts ('Vvedite veshestvennoe chislo: ')
a = gets.to_i
puts ('Vvedite veshestvennoe chislo: ')
b = gets.to_i
puts ('Vvedite veshestvennoe chislo: ')
c = gets.to_i
d = b * b - 4 * a * c
if (d < 0)
    print ('Korney net')
elsif (d == 0)
    print ('1 koren')
else
    print ('2 kornya')
end

File: 2_course/ЯПВУ/Laptev_AV1/Лаптев Александр/LaptevAV-2_3.go
package main
import ("fmt"
        
)

func main() {
    var a float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&a)
    var b float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&b)
    var c float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&c)
    d := b * b - 4 * a * c
    if d < 0 {
        fmt.Print ("Korneyi net")
    }  else if d == 0 {
        fmt.Print ("1 koren")
    } else {
        fmt.Print ("2 Korneyi")
    }
}
